src/mouse_events.cpp

```diff
@@ -1141,4 +1165,19 @@ team & mouse_handler::current_team() {
 }
 
 mouse_handler *mouse_handler::singleton_ = nullptr;
+
+/**  Use this to prevent the mouse to be selecting any unit for
```

Comment:
Remove excessive indent in comment line and move trailing comment closer (`*/`) to its own line.

Reply:
Will comply request.